Box Beautification & Bar Rework & Issue 9706 Fix

KathrinBailey opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->
## About The Pull Request
A facelift for Box Station, many minor changes that stack up to making a better BoxStation©.
Bar and dorms rework with minor changes and beautifications in other places. Adds gas miners also.

**I'm clueless about the map merger stuff and haven't been able to run it (says it needs a module named bidict). Map merger has not been run on the map, so that may be a problem. If I need to do stuff with this then it's probably easier to give someone the map file for the mto run the merger or something.**
<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

## Why It's Good For The Game
Makes the map more beautiful in general. Brings up Box's very old bar that has had no love at all to modern standards. The bar is now closet to a restaurant, it looks much nicer and from the pictures I've put in Discord, people seem to like it. Expands dorms with two 'luxury' rooms that have their own bathroom. And before anyone asks: **No functionality has been lost when making space for the changes. Everything has just been nudged slightly.** This doesn't make Box any 'bigger' either.
What you can do if this gets pushed:
• Make your own space restaurant! You can now, as a gimmick, cooperate as a department to make a restaurant.
• Have enough table space in the bar to seat a group up to 6 people!
• Enjoy downtime in a group in two large luxury cabins!
• Make a fun gimmick room in maint!
• Lock the doors whilst on the toilet, so that pesky borg doesn't barge in!
Anyway, the primary purpose of this PR is to improve the standard of roleplay in the server and to make the map look prettier.

## Changelog
:cl:
add: Two extra 'luxury' dorms rooms!
add: Gas miners to atmos.
add: Posters around the station.
add: Vacant room added to the Starboard Bow with it's own APC, above electrical maintenance.
add: New trendy clothes to the locker room, giving variety and bringing fashion back onto Nanotrasen stations.
add: Coloured bedsheet and towel bin.
add: Maid uniforms for the janitor.
tweak: Completely reworked bar. Milk kegs added in bar office. The bar has been changed for a homey restaurant feel just in time for Christmas! You can now run it as an actual restaurant! Local Bartender Icktsie III loved it so much he rolled around on the new floor smiling happily.
tweak: Dorms rework. Fitness room now has lots of costumes and outfits.
tweak: Junk removed from engineering, welding goggles added.
tweak: Welding tools in engineering replaced with industrial welding tools.
tweak: Package wrappers and hand labellers now in major departments.
tweak: Cell charger moved from engineering lobby to the protolathe room, just like how it is in all of the other maps and just where the cell charger is actually needed.
tweak: Library redesigned to have a private room and a 3x3 private study that is cleaned up.
tweak: Paper bins have gone big or gone home, with premium stationery scattered around. Engineering and security now have a labeller and packaging supplies.
fix: Dark spot top left of Botany fixed.
fix: Huge galactic-sized dark spot in bar fixed.
fix: Light replacers now are less horrifically overpowered and PTSD-inducing for the server.
fix: Fixes issue 9706: https://github.com/Citadel-Station-13/Citadel-Station-13/issues/9706 Part of maint getting hit by radstorms.

_Kathrin's Box Beautification:_
/:cl:

Original changelog (other changes in comments):
**Primary Changes**
• Complete bar rework, featuring a restaurant-like theme. Tablespace increased by 140% with seats with differing capacities to support small and large groups. Fixes the _**huge**_ dark spot in the bar with a central 'pillar' with lights all around. Bar now has wooden flooring all over except behind the bar. There's also a condiments table. Fluff notes and a paper bin have been left behind, there's a noticeboard all to facilitate a restaurant gimmick, should hopefully raise the RP level. One arcade machine of every type added.
• Bar Office cleaned up, the shotgun has been moved to the ID-locked cabinet. Getting a free shotgun round-start with only a welder and wrench is a little odd in low-pop when there is no Bartender. Coffee and cola vending machines removed, the Bartender is next to a giant cola dispenser anyway, kegs are in their place. Waiter uniforms added.
• Bathroom rework, it's now compressed and has bolt buttons for the doors. Each restroom now has a sink, mirror, toilet, bolt switch and light in the same tile, much closer to a real-life restroom. You cannot see through the shower curtain when it is closed now! You also can't see through it when it is open, so that's a trade-off. It was either you can always see or you can't. Privacy is good. The floor is now blue titanium, it has a more bathroomy feel.
• Added two new 'premium' dorms rooms with their own bathrooms. They both have a fireplace and storage.
• Increased the size of the two 'Cabin' rooms from a 3x2 to a 3x3, removed the table and stool on the other old dorm rooms, they just take up space and offer no function.
• Fitness room reworked. It now is a secondary locker room except without the personal closets. This means future mappers can remove/rework the old one if they want. This is because it is next to the holodeck, no one wants to put on holographic clothes, walk out and be naked, or to go into your holo-adventure dressed as an engineer. To make space for this, maint right of dorms has been compressed with no loss of functionality or feature whatsoever.
• All dorms rooms now have numbers and renamed doors to a consistent naming scheme. A washing machine and linen bin has been added
• Removes theatre. No one uses it.

**Secondary Changes**
• Added a small dorm room under the locker room, compressing the locker room showers to save space. No one uses these showers, so this room gives people an opportunity for privacy.
• Added an 'Instrument Storage' room in maint, using existing space. It has instruments and a spare piano, this is done to facilitate more gimmick opportunities.
• Added a new room into maint by deleting unused space to comply with a Discord request for a 'DIY' room in maint. It's at the top right and small.
• Posters added throughout the station, such as a PDA ad and Love Ian posters around the HoP office, cigarette posters in the Detective's office, cola brand posters next to their respective vendors' etcetera. This should add a bit of colour and life to the station.
• Locker Room now has a linen bin, it didn't before for some reason.
• Maid uniforms added to janitor closet. Posters added to janitor closet.
• Added gas miners into atmospherics gas tanks.
• Added two firefoam and high-cap water dispensers into atmos because why not?
• Arrivals now has some catwalks and lattices in space with big bright anchored docking lights that I have on my own map. It just looks pretty.
• Adds a decorative bridge in space to the south of the station to make EVA a little less consistent.
<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->
first pr ever please don't bully

##Images:
https://i.gyazo.com/ef83f7ec616ff5acbdd7c8d549055f0d.png
https://i.gyazo.com/bd4c0e4d1fdb7a4705c91d4630e1e5a4.png
https://i.gyazo.com/cf273a97cb9d64a5d5798dd61e4e4f02.png
https://i.gyazo.com/f9a420340fca30349e49ea94d107a0fd.png
https://i.gyazo.com/48369f96429d6a7b5490e76734c19ac1.png
https://i.gyazo.com/474370cb2949a57c815b2464a1d98bfe.png (Miners added to other tanks too)
https://i.gyazo.com/51f01f9c4f4a2b3203d0fa8932affed6.png 
https://i.gyazo.com/98c775f3dcd764bb62e803ebe8ea5cb8.png (overview pic)

KathrinBailey committed:
Box Rework

Bar, dorms and bathrooms rework, gas miners for atmos.

deathride58 reviewed (dismissed):
Use the mapmerger.

KathrinBailey committed:
map merged

using strong dmm

KathrinBailey committed:
backup box deleted

bye bye old box

KathrinBailey committed:
Everything should be okay now, map merger should have done it right I think?

Varedit objects turned into new instances, which was advised in Discord to fit all the problems with them. Air alarms, APCs and other stuff that were previously in iffy places have been moved. Cell charger moved from engineering lobby to near the protolathe, just like how it is on other maps. Hand labellers and packaging added to service and engineering, other maps give them to botany. Fountain pens for heads and Captain's one for Cap, because, erm, why not?

KathrinBailey committed:
RLD NERF AND BUGFIX BEFORE NEW ROUND AHH SPEEEEED

WILL POST CHANGES IN THE PR SPEEEEED IT FIXES BUGS POSTED IN COMMENTS AAHHH

KathrinBailey committed:
Final commit

All found bugs have been fixed, every major greivence people have had with the changes has been resolved except from the removal of the boxing ring, which I have instead just left space with helpful turf_decals for people to build their own.

There is nothing left to do as far as I'm aware, this should work just fine.

Sishen1542 commented on code/game/objects/items/RCD.dm:
```diff
@@ -663,8 +663,8 @@ RLD
 	righthand_file = 'icons/mob/inhands/equipment/tools_righthand.dmi'
-	matter = 500
-	max_matter = 500
+	matter = 200
```
????

KathrinBailey replied:
You didn't copy and paste this correctly.

The RLD is overpowered and needs nerfing for it to be viable to be used in-game, it probably should be nerfed much harder than this.

Sishen1542 replied:
You're doing something completely unrelated to the map changes... in a map change PR. 
That's not how it works.

JTGSZ commented on code/game/objects/items/RCD.dm:
```diff
@@ -663,8 +663,8 @@ RLD
 	lefthand_file = 'icons/mob/inhands/equipment/tools_lefthand.dmi'
 	righthand_file = 'icons/mob/inhands/equipment/tools_righthand.dmi'
-	matter = 500
-	max_matter = 500
```
Hello, Make a separate PR for this.

KathrinBailey committed:
RLD nerf being made in another pr

yike

KathrinBailey committed:
1

2

KathrinBailey committed:
yikes

yikes 2.0

KathrinBailey committed:
Fixes issue 9706.

Incorrect area on Chapel that results in the tile above the Chapel APC in maintenance getting hit by radstorms.

KathrinBailey committed:
Merge branch 'Kat'sBoxRework' of https://github.com/KathrinBailey/Citadel-Station-13 into Kat'sBoxRework

KathrinBailey committed:
Removed RLDs

RLDs replaced with normal light replacers.
If the only acceptable nerf to the RLD is to rework them I'd rather just remove them than delay this PR by weeks just to learn how to nerf them, since I can spend that time trying to get my own map finished instead.

KathrinBailey committed:
Camera coverage of modified areas should be fine, boxing ring (grudgingly) added.

Kev:
1. bedroom in dorms needs removing
2. needs boxing ring

Both have been done, the map should be ready for another testmerge to see if it is all okay.

KathrinBailey committed:
Fixed Chapel APC + Library edits

Made library look a little nice and fixed duplicated APCs.

KathrinBailey committed:
please please fix merge conflict

KathrinBailey committed:
I hate my life

KathrinBailey committed:
Merge remote-tracking branch 'upstream/master' into Kat'sBoxRework

KathrinBailey committed:
yeah there we go self-confidence destroyed misshun accomplished

KathrinBailey committed:
weh work please no error

error man bad

KathrinBailey committed:
Bugfix from testmerge

Fixes the terrible stupid mistake.
Blood bank generator in medical lathe room on request, it's there on Pubby and heck I suppose it's something for doctors to do.

KathrinBailey committed:
Persi's Sleepers

KathrinBailey committed:
Bugfixes

Fixes incorrect areas
Fixes disconnected air canisters in maint
Adds Persi's sleepers back into perma
Small changes to perma to make it look slightly cleaner. Kink room in perma because it's probably really boring in there.

KathrinBailey committed:
Maaapmeeergeeee?

I forgot last commit. Hopefully this'll do it.

KathrinBailey committed:
Addresses Aelita + Gray + Sylvara, fixes cams, detective office & HoP stamp, room names and airlock cyclelink

Fixes:
• Missing HoP stamp
• Missing Detective handcuffs
• Improper dorms camera names
• Luxury rooms not really mentioned to be luxury
• Missing airlock cyclelink helper on maint south area that was reworked for better EVA spacewalking on the outside.

KathrinBailey committed:
Permabrig broken area fix

oopsie I forgot because I'm always tired fuck 6am morning shifts.
